src/app/components/ThemeProviderSCSSModules/fontSizes.module.scss

```diff
@@ -0,0 +1,92 @@
+    groupB: (font-size: var(--font-size-paragon-group-b), line-height: var(--line-height-paragon-group-b)),
+    groupD: (font-size: var(--font-size-paragon-group-d), line-height: var(--line-height-paragon-group-d)),
+  ),
+  doublePica: (
+    groupA: (font-size: var(--font-size-double-pica-group-a), line-height: var(--line-height-double-pica-group-a)),
+    groupB: (font-size: var(--font-size-double-pica-group-b), line-height: var(--line-height-double-pica-group-b)),
+    groupD: (font-size: var(--font-size-double-pica-group-d), line-height: var(--line-height-double-pica-group-d)),
+  ),
+  greatPrimer: (
+    groupA: (font-size: var(--font-size-great-primer-group-a), line-height: var(--line-height-great-primer-group-a)),
+    groupB: (font-size: var(--font-size-great-primer-group-b), line-height: var(--line-height-great-primer-group-b)),
+    groupD: (font-size: var(--font-size-great-primer-group-d), line-height: var(--line-height-great-primer-group-d)),
+  ),
+  bodyCopy: (
+    groupA: (font-size: var(--font-size-body-copy-group-a), line-height: var(--line-height-body-copy-group-a)),
+    groupB: (font-size: var(--font-size-body-copy-group-b), line-height: var(--line-height-body-copy-group-b)),
+    groupD: (font-size: var(--font-size-body-copy-group-d), line-height: var(--line-height-body-copy-group-d)),
+  ),
+  pica: (
+    groupA: (font-size: var(--font-size-pica-group-a), line-height: var(--line-height-pica-group-a)),
+    groupB: (font-size: var(--font-size-pica-group-b), line-height: var(--line-height-pica-group-b)),
+    groupD: (font-size: var(--font-size-pica-group-d), line-height: var(--line-height-pica-group-d)),
+  ),
+  longPrimer: (
+    groupA: (font-size: var(--font-size-long-primer-group-a), line-height: var(--line-height-long-primer-group-a)),
+    groupB: (font-size: var(--font-size-long-primer-group-b), line-height: var(--line-height-long-primer-group-b)),
+    groupD: (font-size: var(--font-size-long-primer-group-d), line-height: var(--line-height-long-primer-group-d)),
+  ),
+  brevier: (
+    groupA: (font-size: var(--font-size-brevier-group-a), line-height: var(--line-height-brevier-group-a)),
+    groupB: (font-size: var(--font-size-brevier-group-b), line-height: var(--line-height-brevier-group-b)),
+    groupD: (font-size: var(--font-size-brevier-group-d), line-height: var(--line-height-brevier-group-d)),
+  ),
+  minion: (
+    groupA: (font-size: var(--font-size-minion-group-a), line-height: var(--line-height-minion-group-a)),
+    groupB: (font-size: var(--font-size-minion-group-b), line-height: var(--line-height-minion-group-b)),
+    groupD: (font-size: var(--font-size-minion-group-d), line-height: var(--line-height-minion-group-d)),
+  ),
+);
+
+@mixin gel-font-size($scale, $group) {
+  $values: map-get(map-get($font-sizes, $scale), $group);
+  font-size: map-get($values, font-size);
+  line-height: map-get($values, line-height);
+}
```

Comment:
This was a really difficult conversion, the theme dictates the font size and line height so it seemed sensible for these be declared as global variables to set at runtime by JS based on the service theme. In theory the algorithm otherwise works the same as the ts version: https://github.com/bbc/simorgh/blob/latest/src/app/components/ThemeProvider/fontSizes.ts